bridge/opentracing/bridge.go

```diff
@@ -73,17 +73,7 @@ func (c *bridgeSpanContext) ForeachBaggageItem(handler func(k, v string) bool) {
 }
 
 func (c *bridgeSpanContext) IsSampled() bool {
-	return c.otelSpanContext.IsSampled()
-}
-
-// TraceID returns TraceID of underlying span
-func (c *bridgeSpanContext) TraceID() trace.TraceID {
-	return c.otelSpanContext.TraceID()
-}
-
-// SpanID returns SpanID of underlying span
-func (c *bridgeSpanContext) SpanID() trace.SpanID {
-	return c.otelSpanContext.SpanID()
+	return c.SpanContext.IsSampled()
```

Comment:
I think we can remove the definition of `func (c *bridgeSpanContext) IsSampled() bool {`. We get it thanks to type embedding.

Reply:
Removed.